src/_includes/header.liquid

```diff
@@ -17,13 +17,7 @@
           >Events</a
         >
       </li>
-      <li>
-        <a
-          class='nav-item'
-          href='/media'
-          >Media</a
-        >
-      </li>
+      <!-- <li><a class='nav-item' href='/media'>Media</a></li> -->
```

Comment:
Is this temporary? And if you'd like to add again later, would it make sense to add an issue and fully remove it rather than have code left commented out?

Reply:
My personal preference is to remove stuff unless it's planned to go back in, mostly as-is, quite soon. And make sure the work to put it back in is captured in an issue.

But I totally defer to you/Roy for this codebase!

Reply:
Let's remove it - we have it in git history + capturing it as an issue seems the best approach.

Reply:
Done! Ty!